src/engine/BytecodeIterator.v3

```diff
@@ -81,12 +81,17 @@ class BytecodeIterator {
 				HEAP_TYPE,
 				U32,
 				I32,
-				I64 => codeptr.skip_leb();
+				I64,
+				CONT_INDEX => codeptr.skip_leb();
 				VALUE_TYPES => codeptr.skip_value_types();
 				LABELS => {
 					var count = codeptr.read_uleb32();
 					for (i < count + 1) codeptr.skip_leb();
 				}
+				HANDLERS => {
```

Comment:
Are the handlers a list of pairs (tag_index, label), i.e. two LEBs?

Reply:
My bad. This should be fixed in the new commit.